vue-frontend/src/components/Blockly.vue

```diff
@@ -510,7 +516,7 @@ export default {
 
      params.get((res) => {
        this.params = res;
-       setTimeout(this.load_blockly, 500);
+       setTimeout(this.load_blockly, 10);
```

Comment:
Maybe change this to setImmediate, or Vue.nextTick()

Reply:
Tried Vue.nextTick(), but did not work. Maybe we an have a look at that at a later stage.